ios/FluentUI/Drawer/DrawerPresentationController.swift

```diff
@@ -5,6 +5,13 @@
 
 import UIKit
 
+// MARK: - DrawerPresentationDelegate
+
+protocol DrawerPresentationDelegate: AnyObject {
+    /// Called when background view is tapped
+    func backgroundViewTapped()
```

Comment:
protocol should always send the controller as param

Reply:
Updated protocol method signature.